replit_river/session.py

```diff
@@ -503,7 +508,7 @@ async def _remove_acked_messages_in_buffer(self) -> None:
     async def start_serve_responses(self) -> None:
         self._task_manager.create_task(self.serve())
 
-    async def close(self, is_unexpected_close: bool) -> None:
```

Comment:
rofl, we've been providing this field for naught all this time??

Reply:
i think it did something at some point ???